include/moveit_visual_tools/moveit_visual_tools.h

```diff
@@ -323,33 +323,40 @@ class MoveItVisualTools : public rviz_visual_tools::RvizVisualTools
                              const rviz_visual_tools::colors& color = rviz_visual_tools::GREEN);
 
   /**
-   * \brief Create a MoveIt Collision block at the given pose
+   * \brief Create/Publish a MoveIt Collision block at the given pose
```

Comment:
i'd rather you copy this doxygen for reach function. it seems these two functions due fairly different things, and the \brief should explain that difference better.

Reply:
done